### DISCUSSION TITLE
HDR overlay icon when HDR content is playing

### DISCUSSION THREAD

**Discussion by BAXSD** [anddea/revanced-patches#574] [General]
I know that HDR does not work when spoofing client but for those that have no issues with playback and dont spoof client it would be nice to see a HDR icon next to quality/resolution icon. Maybe you could even tap on it and you would get an option to enable or disable HDR when HDR content is playing. Now if you havent read the title/description of the video you wouldnt know that HDR is playing and there is not always info in the title/description that video has HDR. Would this be possible? Is it too much work to make it happen?

I also curious if there will be a fix to play HDR content for those who Spoofs client because of the playback issue?

Anyway I do really appreciate your work @anddea 
I hope you continue your amazing work. Thanks in advance 🥇

**Comment by anddea** [comment]
Never thought of that, so not sure if it's hard but taking even more space of the player seems like a bad UI/UX. I'll think about it.

**Reply by BAXSD** [reply]
> Never thought of that, so not sure if it's hard but taking even more space of the player seems like a bad UI/UX. I'll think about it.

Well you could always have a toggle for it in settings to show or not show. Everyone wins :)
